string model
string test
string[] inputTypes
---
bool result
